plotly/tools.py

```diff
@@ -6208,6 +6207,9 @@ def create_dendrogram(X, orientation="bottom", labels=None,
         if len(s) != 2:
             exceptions.PlotlyError("X should be 2-dimensional array.")
 
+        if distfun is None:
+            distfun = scs.distance.pdist
+
```

Comment:
what is the motivation behind this change? It seems like by removing it from the call signature it is less informative to users about what `distfun` is.